R/nbinomdm.R

```diff
@@ -0,0 +1,79 @@
+#' Negative Binomial Data Maker
+#'
+#' nbinomdm uses rnbinom function from the stats package to create a data frame
```

Comment:
try this description:  
Simulates over-dispersed frequency data using the stats::rnbinom function.